### PR TITLE
Few touch tests + Xbox Controller mappings on Android

### PR DESCRIPTION
**Pull request by todi1856**
Android Game Controllers:
* Added correct map for Xbox Controller + test

Added two touch tests which are currently failing, so we can decide what to do:
* Devices_TouchesWithSameIdDontGetStuck
  two touches with same id happen during one frame, because of that touch is stuck in Stationary phase.
  Such behavior happened on Android, where it was legal for touches to have same id, when they happen one after another
  If we don't allow such behavior, can we add an exception or error in Development build?

* Devices_TouchesWithWrongTimestampCorrectlyRecognized
  Touch event which comes timestamp which is less than previous event is implicitly ignored. Happened on android due precision issues... Again, can input system in development build throw a warning that event is ignored?

### CODE REVIEW

File: Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs [hunk 1]
```diff
@@ -2032,6 +2032,78 @@ public void Devices_TouchesBecomeStationaryWhenNotMovedInFrame()
         Assert.That(device.allTouchControls[0].phase.ReadValue(), Is.EqualTo(PointerPhase.Stationary));
     }
 
+    [Test]
+    [Category("Devices")]
+    public void Devices_TouchesWithSameIdDontGetStuck()
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs]: Please name TODO_Devices_TouchesWithSameIdDontGetStuck. Regular tests should be green.

File: Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs [hunk 2]
```diff
@@ -2032,6 +2032,78 @@ public void Devices_TouchesBecomeStationaryWhenNotMovedInFrame()
...
+        InputSystem.Update();
+        InputSystem.Update();
+
+        Assert.That(device.activeTouches.Count, Is.EqualTo(0));
+        Assert.That(device.allTouchControls[0].phase.ReadValue(), Is.EqualTo(PointerPhase.None));
+    }
+
+    [Test]
+    [Category("Devices")]
+    public void Devices_TouchesWithWrongTimestampCorrectlyRecognized()
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs]: Same here. Prefix with TODO_.

**Review by Rene-Damm — CHANGES_REQUESTED** [review]

File: Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs (hunk 2 above)
Comment (Rene-Damm) [reply Packages/com.unity.inputsystem/Tests/InputSystem/CoreTests_Devices.cs]: Alternatively we could also adopt a FIXME_ prefix for tests that isolate known bugs (unlike the more general "this still needs to be implement" kind of thing that the TODO_ tests generally revolve around).

**Comment by todi1856** [comment]
@Rene-Damm prefixed with FIXME_

**Comment by todi1856** [comment]
Closing for now, found more issues with gamepad

**Pull request closed without merging**